packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2284,8 +2293,8 @@ export default class Meeting extends StatelessWebexPlugin {
         // @ts-ignore - config coming from registerPlugin
         // user need to be joined to start the llm and receive transcription
         if (this.isJoined()) {
-          if (transcribing && !this.transcription && this.config.receiveTranscription) {
```

Comment:
Why is this config removed?

Reply:
Oops. I did it to test something and mistakenly checked in. Will remove